release notes/v0.47.0.md

```diff
@@ -0,0 +1,113 @@
+    {name: 'baz', value: 'qux', url: 'https://grafana.com'},
+]);
+
+const cookies = context.cookies('https://test.k6.io');
+console.log(cookies.length);         // 1
+console.log(cookies[0].name);        // foo
+console.log(cookies[0].value);       // bar
+
+context.clearCookies();
+console.log(context.cookies.length); // 0
+```
+
+### Add support for browser module's `page.on('console')` [browser#1006](https://github.com/grafana/xk6-browser/pull/1006)
+
+Allows users to register a handler to be executed every time the `console` API methods are called from within the page's JavaScript context. The arguments passed into the handler are defined by the [ConsoleMessage](https://k6.io/docs/javascript-api/k6-experimental/browser/consolemessage/) class.
+
+```js
+page.on('console', msg => {
+    check(msg, {
+        'assertConsoleMessageType': msg => msg.type() == 'log',
+        'assertConsoleMessageText': msg => msg.text() == 'this is a console.log message 42',
+        'assertConsoleMessageArgs0': msg => msg.args()[0].jsonValue() == 'this is a console.log message',
+        'assertConsoleMessageArgs1': msg => msg.args()[1].jsonValue() == 42,
+    });
+});
+
+page.evaluate(() => console.log('this is a console.log message', 42));
+```
+
+### UX improvements and enhancements
+
+- [#3338](https://github.com/grafana/k6/pull/3338), [xk6-grpc#48](https://github.com/grafana/xk6-grpc/pull/48) Adds support for the gRPC reflection protocol v1.
+- [#3290](https://github.com/grafana/k6/pull/3290) Adds error logging when executing `setup` and `teardown` via REST API.
+- [#3327](https://github.com/grafana/k6/pull/3327) Adds commit identifier for the k6 build when running `k6 version`.
+- [#3340](https://github.com/grafana/k6/pull/3340) Updates k6 ``*-with-browser`` Docker images to automatically set the `no-sandbox` environment variable.
+- [#3335](https://github.com/grafana/k6/pull/3335) The character limit for metric names increased from 63 to 128 after the OpenTelemetry update. k6 will return an error starting on the next release (v0.48.0) if users hit the limit.
+- [browser#1007](https://github.com/grafana/xk6-browser/pull/1007) Adds a `k6` object (`window.k6 = {};`) to help identify k6 browser module tests.
+- [browser#1022](https://github.com/grafana/xk6-browser/pull/1022) Refactors the `check` in `examples/fillform.js` so that it matches the type definitions and documentation for `check`.
+
+## Bug fixes
+
+- [xk6-grpc#47](https://github.com/grafana/xk6-grpc/pull/47) Fixes the premature closing of a gRPC stream when a stream's client has finished sending. Thanks to @thiagodpf for reporting!
+- [#3344](https://github.com/grafana/k6/pull/3344), [xk6-grpc#49](https://github.com/grafana/xk6-grpc/pull/49) Adds support for Google's protobuf wrappers. Thanks to @zibul444 for reporting!
+- [#3346](https://github.com/grafana/k6/pull/3346) Fixes a race condition when `http.batch` is used with a binary response.
```

Comment:
I'm wondering if this should be in a bug fixes section. If I remember correctly, the bug was caused by the goja dependency update, spotted in the tests and fixed. So maybe it's more about maintenance or improvements or even "squash" with the line about the goja update. 

@mstoykov, what do you think?

Reply:
Let's squash then, I'm going to apply it :+1:

Reply:
I just dropped it as it wasn't an issue in the previous version so it doesn't make sense to mention the fix.